### ISSUE TITLE
A11y: Provide meaningful alternative text to the controls of uploaded a profile picture

### ISSUE THREAD

**Issue by daniellekirkwood** [vector-im/element-web#24919]
<img width="168" alt="Screenshot 2023-03-24 at 11 33 08" src="https://user-images.githubusercontent.com/89144281/227510572-8195000b-e405-4754-8c7a-beed14c261a3.png">

Content that is presented purely graphically is not accessible to blind users.

Suggested solution: A meaningful text alternative that takes the place of the graphic and conveys its content.

**Comment by t3chguy** [comment]
![image](https://user-images.githubusercontent.com/2403652/227511182-a2d00d34-58cc-47ca-b913-bc5102ba69c9.png)

This already uses `aria-labelledby` to link the button with the `Upload` text and the `Remove` text

Is the only complaint here that it doesn't say "Avatar" or anything to that context?

**Comment by t3chguy** [comment]
The text is just `Upload` and `Remove` - which isn't really sufficient for a blind person having 0 visual context
